_rules/iframe-non-empty-accessible-name-cae760.md

```diff
@@ -17,13 +17,17 @@ acknowledgments:
   authors:
     - Jey Nandakumar
     - Wilco Fiers
+    - Tom Brunet
   funding:
     - WAI-Tools
 ---
 
 ## Applicability
 
-This rule applies to `iframe` elements that are [included in the accessibility tree][] and do not have a negative `tabindex` [attribute value][].
+This rule applies to `iframe` elements that are [included in the accessibility tree][] and for which all of the following are true:
+- does not have a negative `tabindex` [attribute value][]; and
+- does not have a `role` [attribute value][] of `presentation`; and
+- does not have a `role` [attribute value][] of `none`.
 
 **Note:** `frame` element is deprecated, this rule does not consider `frame` or `frameset` elements.
```

Comment:
Would you mind moving this note to the background? That feels like a better place for it.